python/pyspark/sql/functions.py

```diff
@@ -55,11 +55,13 @@
 
 # Keep pandas_udf and PandasUDFType import for backwards compatible import; moved in SPARK-28264
 from pyspark.sql.pandas.functions import pandas_udf, PandasUDFType  # noqa: F401
+
+# TODO: should directly rename the internal functions
```

Comment:
It's actually fine to don't rename. In practice, we only name underscore when the file is exposed as an API. `pyspark.sql.utils` isn't documented so it's fine. If those methods are used in other (internal) modules, we can just don't add `_` prefix.

Reply:
got it, let me remove it